README.md

```diff
@@ -37,26 +22,29 @@ https://dev.futurice.com:8081/vimma/
 
 ## Running on your machine:
 
-Create a Python3 virtual enviroment and use it for most commands.
```

Comment:
We can move to the bottom. The point is to move away from "doing everything by hand".